src/avfoundation.cs

```diff
@@ -12571,6 +12707,29 @@ interface AVRouteDetector {
 		bool MultipleRoutesDetected { get; }
 	}
 
+	interface IAVCapturePhotoFileDataRepresentationCustomizer {}
+	[NoWatch, NoTV, NoMac, iOS (12,0)]
+	[Protocol]
+	interface AVCapturePhotoFileDataRepresentationCustomizer
+	{
+		[Export ("replacementMetadataForPhoto:")]
+		[return: NullAllowed]
+		NSDictionary<NSString, NSObject> GetReplacementMetadata (AVCapturePhoto photo);
+
+		[Export ("replacementEmbeddedThumbnailPixelBufferWithPhotoFormat:forPhoto:")]
+		[return: NullAllowed]
+		CVPixelBuffer GetReplacementEmbeddedThumbnailPixelBuffer ([NullAllowed] out NSDictionary<NSString, NSObject> replacementEmbeddedThumbnailPhotoFormatOut, AVCapturePhoto photo);
```

Comment:
not sure we need the `PixelBuffer` suffix since it's already present with the return type **and** thumbnail already hints there's an image coming out